Drush role:create is broken

**The bug / steps to reproduce**
When I ran `$ drush role:create 'some role'` I got:
```
 [error]  AssertionError: Role label is expected to be a string. in assert() (line 851 of /Users/me/project/web/core/modules/user/user.module) #0 /Users/me/project/web/core/modules/user/user.module(851): assert(false, 'Role label is e...')`
 ```
And then a bunch of debugging info (stack trace). 

After this, I could not see the new role in the admin UI. But when trying the command again (this time with two arguments, as should also work), it said:

```
In EntityStorageBase.php line 557:

  'user_role' entity with ID 'some role' already exists.
  ``` 

**Expected behavior**
The new role is created and available in the admin UI.

**Actual behavior**
See above.

**Workaround**
Create the role from the admin UI.

### System Configuration
| Q               | A
| --------------- | ---
| Drush version?  | 11.0.7
| Drupal version? | 9.3.9
| PHP version     | 8.0.12
| OS?             | MacOS

**Additional information**
This happened to me when following the instructions at https://next-drupal.org/learn/quick-start. A couple of contrib modules and two patched had already been installed at this point, but I don't think that is causing the problem.

--- a/src/Drupal/Commands/core/RoleCommands.php
+++ b/src/Drupal/Commands/core/RoleCommands.php
@@ -21,8 +21,6 @@ class RoleCommands extends DrushCommands implements SiteAliasManagerAwareInterfa
      * @command role:create
      * @param $machine_name The symbolic machine name for the role.
      * @param $human_readable_name A descriptive name for the role.
-     * @usage drush role:create 'test role'
-     *   Create a new role 'test role'. On D8, the human-readable name will be 'Test role'.
      * @usage drush role:create 'test role' 'Test role'
      *   Create a new role with a machine name of 'test role', and a human-readable name of 'Test role'.
      * @aliases rcrt,role-create
@@ -31,7 +29,7 @@ class RoleCommands extends DrushCommands implements SiteAliasManagerAwareInterfa
     {
         $role = Role::create([
         'id' => $machine_name,
-        'label' => $human_readable_name,
+        'label' => $human_readable_name ?: ucfirst($machine_name),
         ], 'user_role');
         $role->save();
         $this->logger()->success(dt('Created "!role"', ['!role' => $machine_name]));

